rest-api-spec/src/main/resources/rest-api-spec/api/indices.get_template.json

```diff
@@ -34,7 +34,7 @@
       },
       "master_timeout":{
         "type":"time",
-        "description":"Explicit operation timeout for connection to master node"
+        "description":"Specify timeout for connection to master"
```

Comment:
I forgot to deprecate the parameter here..

Reply:
Updated in commit https://github.com/opensearch-project/OpenSearch/pull/2660/commits/b914741f9683edc583b7ca6ebc0b5615db08d248